app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -111,4 +112,25 @@ def create_network_router_queue(userid, options = {})
     }
     MiqTask.generic_action_with_callback(task_opts, queue_opts)
   end
+
+  def create_security_group(options)
+    SecurityGroup.raw_create_security_group(self, options)
+  end
+
+  def create_security_group_queue(userid, options = {})
```

Comment:
@gildub yeah, we will have to do more refactoring though, all the models needs to be consistent (base class no raw methods, raw methods and supports without suffix )